xml/System.Net.Security/SslStream.xml

```diff
@@ -3453,9 +3453,17 @@ Authentication has not occurred.</exception>
         <ReturnType>System.Net.Security.TlsCipherSuite</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <summary>Gets the cipher suite resolved to use by this <see cref="T:System.Net.Security.SslStream"/>.</summary>
+        <value>The cipher suite resolved to use by this <see cref="T:System.Net.Security.SslStream"/>.</value>
```

Comment:
```suggestion
        <value>One of the enumeration values that identifies the cipher suite which was negotiated for this connection.</value>
```

https://github.com/dotnet/dotnet-api-docs/wiki/Property-Value says `One of the enumeration values that <additional information>.` (and then some stuff about defaults that don't apply here)